react/src/Analyses/index.tsx

```diff
@@ -621,6 +669,14 @@ export default function Analyses() {
                                 });
                             },
                         },
+                        {
+                            icon: Delete,
+                            tooltip: "Delete Analysis",
+                            onClick: (event, rowData) => {
+                                setActiveRows(rowData as Analysis[]);
+                                setConfirmDelete(true);
+                            },
+                        },
```

Comment:
Can you hide this if the user is not an admin?

Reply:
On it!

Reply:
Sorry, I meant to physically hide the button, which is done on the datasets page with its delete button.